docs/features/ems_reporting.md

```diff
@@ -53,15 +53,15 @@ For Cumulus instances running on NGAP, submit a [NGAP service desk ticket](https
 
 ### 3) Send public key to EMS. Lambda will provide private key when sftp files to EMS
 
-Upload the corresponding private key to s3, use `{{system_bucket}}` as bucket name and `{{prefix}}/crypto/ems-private.pem` as key,  `{{system_bucket}}` and `{{prefix}}` are configured in `app/config.yml`.  If a different private key file name other than `ems-private.pem` is used, specify it in the `ems` configuration in `app/config.yml`.
+Upload the corresponding private key to s3, use `system_bucket` as bucket name and `{prefix}/crypto/ems-private.pem` as key,  `system_bucket` and `prefix` are configured in your deployment's `terraform.tfvars` file.  If a different private key file name other than `ems-private.pem` is used, specify it in the `ems_private_key` configuration in `terraform.tfvars`.
```

Comment:
In the other document, Mark and I removed all the `{}`, and use '<>' for variables such as `<prefix>/crypto/ems-private.pem`.
https://raw.githubusercontent.com/nasa/cumulus/master/docs/additional-deployment-options/cmr_launchpad_authentication.md

Reply:
@markdboyd  we've not used "<>" elsewhere (even in this PR for that matter), are we trying to avoid braces *entirely*?

Reply:
I mostly feel strongly about removing double braces `{{ }}`. In terms of where we are trying to demonstrate that a value should be replaced, I don't have a strong preference, though as Jenny said I suggested using `<>` in the other document. I'm more than happy if we use `{}` here and change it in the CMR docs

Reply:
I don't mind removing `{{` and `}}` as they're clearly handlebars/mustache/kes, but `{}` is used for interpolation in lots of things and implies it more so than a "<" tag, _to me_. 

I think this might be one that we push off to a 'review docs for consistency' and make our current updates one thing.     On a 1-10 scale I'm at about a 5 on this particular convention.

Reply:
I'm fine with using that convention and reviewing for consistency later